files/es/conflicting/learn/javascript/objects/classes_in_javascript/index.md

```diff
@@ -8,7 +8,7 @@ original_slug: Learn/JavaScript/Objects/Object-oriented_JS
 
 Con lo básico fuera del camino, nos enfocaremos en Javascript Orientado a Objetos (JSOO) — este artículo presenta una descripción básica de la teoría de la Programación Orientada a Objetos (POO), luego explora cómo Javascript emula classes de objetos via funciones constructoras, y cómo crea instancias de objetos.
 
-| Prerequisitos: | Conocimientos básicos de computación, entendimiento básico de HTML y CSS, familiaridad con las bases de Javascript (ver [Primeros pasos con JavaScript](/es/docs/Learn/JavaScript/First_steps) y [Bloques de construcción JavaScript](/es/docs/Learn/JavaScript/Building_blocks)) y las bases de JSOO (ver [Introducción a objetos](/es/docs/Learn/JavaScript/Objects/Basics)). |
+| Prerequisitos: | Conocimientos básicos de computación, entendimiento básico de HTML y CSS, familiaridad con las bases de Javascript (ver [Primeros pasos con JavaScript](/es/docs/conflicting/Learn_web_development/Core/Scripting) y [Bloques de construcción JavaScript](/es/docs/Learn_web_development/Core/Scripting)) y las bases de JSOO (ver [Introducción a objetos](/es/docs/Learn_web_development/Core/Scripting/Object_basics)). |
```

Comment:
@caugner is it intentional to point to `{locale}/conflicting/` URLs?

Reply:
That's weird, but there is a redirect set up:

https://github.com/mdn/translated-content/blob/4f2104fc1ef39780c8ce8260021292d22ebba6ee/files/es/_redirects.txt#L1327

Added via https://github.com/mdn/translated-content/commit/82f53f6ffe2654ab442fb8d4ceaee4211ca90f7f.

Reply:
Thank you so much to both for your help! So we keep the `conflicting` urls in this PR?

Reply:
@Graywolf9 If you look [here](https://developer.mozilla.org/es/docs/Learn_web_development/Extensions/Advanced_JavaScript_objects/Classes_in_JavaScript) on production, you will see that "[Primeros pasos](https://developer.mozilla.org/es/docs/conflicting/Learn_web_development/Core/Scripting)" indeed links to `conflicting/`, so this change to `conflicting` here just makes transparent what currently happens on production.

Since https://developer.mozilla.org/es/docs/Learn_web_development/Core/Scripting exists, you'll probably want to remove `conflicting/Learn_web_development/Core/Scripting` and redirect it back to `Learn_web_development/Core/Scripting`.

(I would recommend to do this in a different branch though, as this `content-autofix-es` branch will be overridden daily.)

Reply:
@caugner I guess that means we should merge and the `es` team can fix `conflicting/*` links in a follow-up?

Reply:
Great! Thanks @caugner and @bsmth , so this PR goes on and then we fix the conflicting files